scripts/presubmit-tests.sh

```diff
@@ -167,7 +167,7 @@ function default_build_test_runner() {
   go build -v ./... || failed=1
   # Get all build tags in go code (ignore /vendor)
   local tags="$(grep -r '// +build' . \
-      | grep -v '^./vendor/' | cut -f3 -d' ' | sort | uniq | tr '\n' ' ')"
+      | grep -v '^./vendor/' | grep -v '^./third_party/' | grep -v "+build tools" | cut -f3 -d' ' | sort | uniq | tr '\n' ' ')"
```

Comment:
@wlynch why the `grep -v "+build tools"` ?

Reply:
This is to exclude packages containing binary dependencies, as specified by https://github.com/golang/go/wiki/Modules#how-can-i-track-tool-dependencies-for-a-module.

Added a comment to document.

Reply:
Ah nice, thanks :+1: